docs/UserGuide.md

```diff
@@ -138,23 +138,24 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Deleting a person : `delete`
+### Deleting an event : `delete`
 
-Deletes the specified person from the address book.
+Deletes the event from the event list.
 
-Format: `delete INDEX`
+Format: `edelete eid/EVENT_ID`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* Deletes the event at the specified `id`.
+* The id refers to the index number shown in the displayed event list.
+* The id **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `event list` followed by `edelete 2` deletes the 2nd event in the event list.
+* `find Beach cleaning` followed by `edelete 1` deletes the 1st event in the results of the `find` command (tentative feature)
+
 
 ### Clearing all entries : `clear`
```

Comment:
Maybe this command should be eclear as there is a vclear command for volunteers too